qds_sdk/commands.py

```diff
@@ -977,6 +977,10 @@ class DbExportCommand(Command):
                          help="Modes 1 and 2: DbTap Id of the target database in Qubole")
     optparser.add_option("--db_table", dest="db_table",
                          help="Modes 1 and 2: Table to export to in the target database")
+    optparser.add_option("--use_customer_cluster", dest="use_customer_cluster", default=False,
```

Comment:
The test cases have been added in next commit. Regarding the api behaviour:
1) It throws an error if use_customer_cluster is True but customer_cluster_label not set.
2) It accepts the request if use_customer_cluster is False but the customer_cluster_label is set but doesn't use the cluster. 
The following were discussed with @msumit and @sourabh912 .